backend/dal/Migrations/v01.07.02/Up/PostUp/01-ViewProperties.sql

```diff
@@ -0,0 +1,213 @@
+PRINT 'Updating View_Properties - Updating Assessment'
+GO
+-- This script provides a way to union both parcels and buildings into a single result of properties.
+-- Updating view to support many-to-many parcels and buildings.TEMPORARY
+CREATE OR ALTER VIEW dbo.[View_Properties] AS
+SELECT
+    p.[Id]
+    , p.[RowVersion]
+    , [PropertyTypeId] =  (SELECT CASE WHEN sp.[ParcelId] IS NOT NULL THEN 2 ELSE 0 END)
```

Comment:
I thought we were going to add `Property.PropertyType`.  Which would allow for this value to be extracted from that property.  I don't think I clearly articulated this in any story however :(.  Might need to be thought through and considered though.